pkg/util/names/names.go

```diff
@@ -16,12 +16,10 @@ const (
 	// - kube-public
 	// - kube-node-lease
 	KubernetesReservedNSPrefix = "kube-"
-	//NamespaceKarmadaSystem is reserved namespace
+	// NamespaceKarmadaSystem is reserved namespace
 	NamespaceKarmadaSystem = "karmada-system"
-	//NamespaceKarmadaCluster is reserved namespace
+	// NamespaceKarmadaCluster is reserved namespace
 	NamespaceKarmadaCluster = "karmada-cluster"
-	//NamespaceDefault is reserved namespace
-	NamespaceDefault = "default"
```

Comment:
The `const()` block used to define reserved namespaces,  people could get the whole list from here when they read code.
So, I don't think it's worth to remove `default` from here and replace by a reference.

Reply:
You're right, I hadn't thought about that.